content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
 
-[1]: /developers/metrics/agent_metrics_submission/?tab=rate
-[2]: /api/?lang=python#post-timeseries-points
 {{% /tab %}}
-{{% tab "GAUGE" %}}
-
-**The `GAUGE` metric submission type represents a value of a particular thing reporting continuously over time.** It is a snapshot of the last recorded value of a particular thing during a defined time interval (flush interval). A `GAUGE` can be used to represent the temperature or memory usage.
+{{< /tabs >}}
 
-For instance, say that the `number.of.requests.gauge` metric is reported every 30 seconds to Datadog with the `GAUGE` type in `server:web_1`.
+### Example
 
-Each value/data point represents the total number of requests received at a point in time. The metric then reports the following values:
+{{< tabs >}}
+{{% tab "COUNT" %}}
 
-* `10` for the first 30 seconds
-* `30` for the second interval of 30 seconds (10 + 20 requests)
-* `30` for the last interval of 30 seconds (no new request are received so the value stays the same)
+Suppose you’re submitting a COUNT metric , `activeusers.basket_size`, from a single host running the Datadog Agent. This metric emits the following values in a 10s time interval: `[1,1,1,2,2,2,3,3]`.
 
-When graphed, this `GAUGE` metric looks like the following:
+The Agent adds all the values received in one time interval and submits the total number, in this case `15`, as the COUNT metric’s value.
 
-{{< img src="developers/metrics/types/gauge_metric.png" alt="Gauge Metric" >}}
+{{% /tab %}}
+{{% tab "RATE" %}}
 
-Discover how to submit gauge metrics:
+Suppose you’re submitting a RATE metric , `queue_messages.rate`, from a single host running the Datadog Agent. This metric emits the following values in a 10s time interval: `[1,1,1,2,2,2,3,3]`.
 
-* [With a custom Agent Check][1]
-* [With DogStatsD][2]
-* [With the Datadog API][3]
+The Agent adds all the values received in one time interval and submits the total number divided by the total number of seconds in this time interval, in this case `1.5`, as the RATE metric’s value.
 
-[1]: /developers/metrics/agent_metrics_submission/?tab=gauge
-[2]: /developers/metrics/dogstatsd_metrics_submission/#gauge
-[3]: /api/?lang=python#post-timeseries-points
 {{% /tab %}}
-{{% tab "HISTOGRAM" %}}
+{{% tab "GAUGE" %}}
 
-**The `HISTOGRAM` metric submission type allows you to measure the statistical distribution of a set of values**. Datadog's `HISTOGRAM` metric type is an extension of the [StatsD timing metric type][1]: it aggregates (on the Agent-side) the values that are sent during a defined time interval (the default flush interval is 10s) and produces different timeseries representing the different aggregations possible for the set of values. Depending on the aggregation, the metric type stored by Datadog is different.
+Suppose you’re submitting a GAUGE metric , `temperature`, from a single host running the Datadog Agent. This metric emits the following values in a 10s time interval: `[71,71,71,71,71,71,71.5]`.
```

Comment:
Suppose you’re submitting a GAUGE metric , `temperature`, from a single host running the Datadog Agent. This metric emits the following values in a flush time interval: `[71,71,71,71,71,71,71.5]`.